api/v1/tetragon/codegen/eventcache/eventcache.pb.go

```diff
@@ -8,8 +8,8 @@ package eventcache
 import (
 	fmt "fmt"
 	tetragon "github.com/cilium/tetragon/api/v1/tetragon"
-	metrics "github.com/cilium/tetragon/pkg/metrics"
 	errors "github.com/cilium/tetragon/pkg/metrics/errors"
+	eventcache "github.com/cilium/tetragon/pkg/metrics/eventcache"
```

Comment:
It would be nice to have a different package name here (`evcachemetrics`?) but the code is generated so it does not matter much.

Reply:
Eh, I think you've got a point. They should probably all be renamed to have metrics as a suffix. I'll do it.